src/common/LanguageSelector/LanguageSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { useTranslation } from 'react-i18next';
+import { Input, Wrapper } from './LanguageSelector.style';
+
+export default function LanguageSelect() {
+  const [t] = useTranslation();
+  const [lang, setLang] = useState(localStorage.getItem('lang') || 'pl');
```

Comment:
I guess we should retrieve this value from i18n provider. Please, check is it possible.

Reply:
you are right. Fixed